src/main/java/racingcar/view/InputValidator.java

```diff
@@ -0,0 +1,62 @@
+package racingcar.view;
+
+import static java.text.MessageFormat.format;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class InputValidator {
+
+    private static final String INVALID_NAME_MESSAGE = "차의 이름은 {0}자 이상 ~ {1}자 이하여야 합니다.";
+    private static final int NAME_LENGTH_LOWER_BOUND = 1;
+    private static final int NAME_LENGTH_UPPER_BOUND = 5;
+    private static final String DELIMITER = ",";
+    private static final int SPLIT_LIMIT = -1;
+    private static final String INVALID_COUNT_MESSAGE = "횟수는 {0}이상의 정수만 가능합니다.";
+    private static final int COUNT_LOWER_BOUND = 1;
+
+    public List<String> validateNames(final String input) {
+        List<String> names = generateNames(input);
+        for (String name : names) {
+            validateName(name);
+        }
+        return names;
+    }
+
+    private List<String> generateNames(final String input) {
+        return Arrays.stream(input.split(DELIMITER, SPLIT_LIMIT))
+                .map(String::trim)
+                .collect(Collectors.toList());
+    }
+
+    private void validateName(final String name) {
+        if (name.length() < NAME_LENGTH_LOWER_BOUND || name.length() > NAME_LENGTH_UPPER_BOUND) {
```

Comment:
또 꺽쇠 방향을 바꾸어 줌으로써 읽는 것과 동시에 해석될 수 있는 문장으로 만들 수 있겠습니다. 
```suggestion
        if (name.length() < NAME_LENGTH_LOWER_BOUND || NAME_LENGTH_UPPER_BOUND < name.length()) {
```

Reply:
꺽쇠 하나로 가독성이 높아지는 것 같아요 대박 👍👍